api/RTCIceCandidateInit.json

```
@@ -67,10 +67,12 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "22",
+              "notes": "The <code>candidate</code> property became optional in a mid-2017 specification update. Firefox implements this change in version 68."
```

For this note, I'd like to lead with the information important to web developers (which version changed) and follow with the detail about the specification. How about something like this?

```suggestion
              "notes": "Before Firefox 68, the <code>candidate</code> property is required. The property became optional in a mid-2017 specification update."
```

---

Yep.